pkgs/applications/science/biology/mmseqs2/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, xxd
+, perl
+, enableAvx2 ? stdenv.hostPlatform.avx2Support
+, enableSse4_1 ? stdenv.hostPlatform.sse4_1Support
+, enableMpi ? (false && !stdenv.isDarwin)
```

Comment:
What's the idea here? The default will always evaluate to false.

Reply:
I wanted it to be false when the target is darwin, but I should use the variable as "enablaMpi && !stdenv.isDarwin" when calling.
Also, I misunderstood that MPI is not available in darwin, so I will change it to merely false.